### PR TITLE
Run tests on GHA

### PR DESCRIPTION
**Pull request by icemac**
* Add support for Python 3.9.
* Update to isort >= 5.
* Test against Zope 5 where possible.
* Built with https://github.com/zopefoundation/meta/pull/57

Tests for Python 3.9 currently fail because of https://github.com/zopefoundation/Products.GenericSetup/pull/99. When it is merged and released tests should get green here, too.

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -1,55 +1,89 @@
...
-basepython = python3.6
+allowlist_externals =
+    mkdir
 commands_pre =
     mkdir -p {toxinidir}/parts/flake8
 commands =
-    isort --check-only --diff --recursive {toxinidir}/src setup.py
-    - flake8 --format=html --doctests src setup.py
-    flake8 --doctests src setup.py
+    - isort --check-only --diff {toxinidir}/src setup.py
```
Comment (sallner) [inline tox.ini:33]: Where does this and the following line come from? I cannot deduce those from the zopefoundation/meta changes. Do we need the `-` here?

File: tox.ini [hunk 2]
```diff
@@ -1,55 +1,89 @@
...
+    - flake8 --format=html src setup.py
+    flake8 src setup.py
```
Comment (sallner) [inline tox.ini:35]: It is somewhat inconsistent to use `{toxinidir}/src setup.py` before and now only relative paths. Did this come up on purpose?

**Review by sallner — COMMENTED** [review]
Some smaller comments and we need to get Python 3.9 working of course.

File: tox.ini (hunk 1 above)
Comment (sallner) [reply tox.ini:33]: Ah, okay I found the place. https://github.com/zopefoundation/meta/pull/57/files#diff-e9d7c2feed27afda348a43f05d3abbe05a27e0fc9d4a0149c3c6ba848be84e56R33

File: buildout.cfg [hunk 3]
```diff
@@ -1,6 +1,6 @@
 [buildout]
 extends =
-    https://zopefoundation.github.io/Zope/releases/4.x/versions.cfg
+    http://zopefoundation.github.io/Zope/releases/master/versions.cfg
```
Comment (jugmac00) [inline buildout.cfg:3]: Is the usage of http instead of https intended?

File: tox.ini (hunk 2 above)
Comment (icemac) [reply tox.ini]: I cleaned it up to make it constantly using `{toxinidir}`.

File: buildout.cfg (hunk 3 above)
Comment (icemac) [reply buildout.cfg:3]: Good catch of a copy-paste-mistake.

**Review by dataflake — APPROVED** [review]

**Comment by icemac** [comment]
Thank you for reviewing this PR. 😃

**Pull request merged**